compiler/luci/pass/src/SubstituteStridedSliceToReshapePass.test.cpp

```diff
@@ -208,6 +208,33 @@ TEST_F(SubstituteStridedSliceToReshapeTest, with_large_end_mask)
   ASSERT_EQ(new_shape->at<loco::DataType::S32>(1), 9);
 }
 
+TEST_F(SubstituteStridedSliceToReshapeTest, with_wrong_begin_end_mask_with_shrink_axis_mask)
+{
+  buildGraph({1, 1, 20}, // input shape
+             {1, 0, 0},  // begin with -1 at 0th dim
```

Comment:
I'd like to use -1 value as in reference model, but buildGraph takes only uint32_t

Reply:
> ... but buildGraph takes only uint32_t

You may have to revise all related codes in `SubstituteStridedSliceToReshapeTest` to accept `int32_t`

Reply:
> > ... but buildGraph takes only uint32_t
> 
> You may have to revise all related codes in `SubstituteStridedSliceToReshapeTest` to accept `int32_t`

I am fine with current test. I will fix the comment.